include/zephyr/bluetooth/audio/pacs.h

```diff
@@ -17,15 +17,15 @@ extern "C" {
 #endif
 
 /** @brief Audio Capability structure. */
-struct bt_audio_capability {
+struct bt_pacs {
```

Comment:
Since are changing the namespace here, we may want to consider Add "Published Audio Capabiltiies" instead of "Audio Capability" in the documentation.

Reply:
```suggestion
struct bt_pacs_capabilities {
```

I think even the name should be changed, to me it otherwise signals being a struct for the complete service.

Reply:
@MariuszSkamra There are more cases where we may want to add "Published", e.g. on Like 29-36. Github won't let me comment on those lines, so can't point them out myself :)

Reply:
done